docs/devguide/contributing.asciidoc

```diff
@@ -56,7 +55,7 @@ simply set it to the `go` directory in your home (`export GOPATH=$HOME/go`).
 --------------------------------------------------------------------------------
 mkdir -p ${GOPATH}/src/github.com/elastic
 cd ${GOPATH}/src/github.com/elastic
-git clone https://github.com/elastic/beats.git
+git clone git@github.com:elastic/beats.git
```

Comment:
I would keep `https` here as it's a public repo and I personally would recommend to pull it over https and only the clone over git by default. Like this you never push accidentally to the elastic repo but always work with PR's.

Obviously if someone needs to create branches / tags this is different but very few contributors need to do this.

Reply:
sounds good, reverting that change
Thanks!